src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Automatic anomaly detection 
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how automatic automatic anomaly detection in New Relic notifies you of unusual app behavior. 
```

Comment:
Double use of automatic in the metaDescription

Reply:
done